Framework/WorkflowAlgorithms/src/ProcessIndirectFitParameters.cpp

```diff
@@ -82,6 +89,14 @@ std::vector<T> getColumnValues(Column const &column, std::size_t startRow,
   return values;
 }
 
+std::vector<double> getNumericColumnValuesOrIndices(Column const &column,
+                                                    std::size_t startRow,
+                                                    std::size_t endRow) {
+  if (column.isNumber())
+    return getColumnValues<double>(column, startRow, endRow);
+  return getIncrementingSequence(0.0, endRow - startRow + 1);
```

Comment:
If startRow > endRow this will produce undefined behaviour.

You could do:
auto const length = startRow >= endRow ? 1 + startRow - endRow : 0;

Then pass length to getIncrementingSequence